scipy/spatial/transform/rotation.pyx

```diff
@@ -878,22 +878,30 @@ cdef class Rotation(object):
         Parameters
         ----------
         seq : string
-            Specifies sequence of axes for rotations. Up to 3 characters
+            Specifies the order of the sequence of
+            rotations to be performed. Up to 3 characters
             belonging to the set {'X', 'Y', 'Z'} for intrinsic rotations, or
             {'x', 'y', 'z'} for extrinsic rotations. Extrinsic and intrinsic
             rotations cannot be mixed in one function call.
         angles : float or array_like, shape (N,) or (N, [1 or 2 or 3])
+            where the number of rows, N, indicates the number of rotation 
+            sets and the columns, ([1, 2 or 3]), provide rotations angles about the 
+            z, y and x axes respectively for each set.
+            
             Euler angles specified in radians (`degrees` is False) or degrees
             (`degrees` is True).
-            For a single character `seq`, `angles` can be:
+            
+            For rotation about a single axis, a single character `seq`
+            indcates the axis, and `angles` can be:
```

Comment:
typo "indcates"

The CI doc failure is real, albeit a bit cryptic: `scipy/spatial/transform/__init__.py:docstring of scipy.spatial.transform.Rotation.from_euler:100: WARNING: Definition list ends without a blank line; unexpected unindent.`

Reply:
I appreciate the help - I clearly have a lot to learn about scipy documentation formatting and the nightly build checks. 

I am still not sure I've fixed the problem (after 5 tries) as now checks are failing against other parts of the auto-build process that I can't imagine that my edits to the doc-string have affected. It's not clear to me if that's because the version I forked is now behind the current version and I need to pull from it or if this is something that is failing for others and will get resolved? Any suggestions are welcome.